mmv1/templates/validator/resource_converters.go.erb

```diff
@@ -54,6 +54,7 @@ func ResourceConverters() map[string][]ResourceConverter {
 		"google_bigquery_dataset_iam_binding":             {resourceConverterBigqueryDatasetIamBinding()},
 		"google_bigquery_dataset_iam_member":              {resourceConverterBigqueryDatasetIamMember()},
 		"google_bigquery_table":                           {resourceConverterBigQueryTable()},
+    "google_redis_instance":                           {resourceConverterRedisInstance()},
```

Comment:
It looks like the indentation here doesn't match (probably because spaces vs tabs)

Reply:
indeed, it was because of spaces and tabs. Fixed in the latest push.